### PR TITLE
Adding butterfly formula

### PR DESCRIPTION
**Pull request by mmcrockett**
@fabiocarvalho777 - Please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+* [Homebrew](https://github.com/Homebrew/brew)
+* Java 7+
+
+## Installation
+
+Run the following in your command-line:
+
+```sh
+$ brew install paypal/butterfly/butterfly
```
Comment (fabiocarvalho777) [inline README.md:15]: Was not this supposed to be `brew install paypal/butterfly`?

**Comment by fabiocarvalho777** [comment]
Now that we are adding this here, does it mean we can remove the `Formula` folder in butterfly repo?

File: README.md (hunk 1 above)
Comment (mmcrockett) [reply README.md:15]: Sorry, not sure. Think we either have to have two commands - 'brew tap paypal/butterfly' then 'brew install paypal/butterfly' or can use single command but have to specify exact formula.

**Comment by mmcrockett** [comment]
Yes, remove Formula from butterfly repo and likely reference the paypal/homebrew-butterfly/README.md  as the brew installer instructions from butterfly.

**Pull request merged**